### PR TITLE
Added text on generating CIDs.

### LINKED ISSUE #12: Use of multiple quic connection on the same port/5-tuple
Write section to provide guidance; maybe in the same section than guidance in port selection in general

### PR DESCRIPTION
**Pull request by martinduke**
Tries to resolve #12. I resisted the urge to reference the QUIC-LB draft, which isn't adopted.

The major substantive change is that I removed the prohibition on non-routing information in the CID. I'm happy to discuss this if people have object.

### CODE REVIEW

**Comment by britram** [comment]
LGTM. will go ahead and land this, then fix up the lint issues.

File: draft-ietf-quic-applicability.md [hunk 1]
```diff
@@ -287,40 +287,44 @@ information is intended to be used by the network. QUIC has a long header that
...
 Connection ID.
 
 ## Server-Generated Connection ID
 
 QUIC supports a server-generated Connection ID, transmitted to the client
 during connection establishment (see Section 6.1 of {{!QUIC}}). Servers behind
-load balancers should propose a Connection ID during the handshake, encoding
-the identity of the server or information about its load balancing pool, in
-order to support stateless load balancing. Once the server generates a
+load balancers may need to  propose a Connection ID during the handshake,
```
Comment (ianswett) [inline draft-ietf-quic-applicability.md]: nit: propose a -> change the 

Also, there's an extra space after 'to'

**Comment by britram** [comment]
@ianswett missed this during the inconsistency, will fix in working copy

**Comment by britram** [comment]
(this is done)

**Comment by martinduke** [comment]
BTW, this was meant to address #29

**Pull request merged**